wavefront_pyformance/runtime_metrics.py

```diff
@@ -0,0 +1,188 @@
+"""Python Runtime Metrics Collection Class."""
+# pylint: disable=E0012,R0205
+
+import gc
+import multiprocessing
+import os
+import resource
+import threading
+
+import psutil
+
+
+class RuntimeCollector(object):
+    """Python Runtime Metrics Collection Class."""
+
+    def __init__(self, registry=None):
```

Comment:
inside init method
you should report hearbeat metrics
see example - 
https://github.com/wavefrontHQ/wavefront-runtime-sdk-jvm/blob/32f51d3b548a937f72905a85efe245090bcd7c61/src/main/java/com/wavefront/sdk/appagent/jvm/reporter/WavefrontJvmReporter.java#L42